src/components/Cards/CardLarge/CardLarge.scss

```diff
@@ -0,0 +1,54 @@
+@import 'styles/utilities/colors.scss';
+@import 'styles/fonts.scss';
+@import 'styles/utilities/typography.scss';
+
+$card-image-height: 330px;
+
+.card-large {
+  display: grid;
+  grid-template-areas: 'image' 'text' 'button';
+  grid-template-rows: $card-image-height auto 74px;
+  overflow: hidden;
+  border-radius: 5px;
```

Comment:
Ive seen this a couple times now - lets add this to a util file so we can use it in multiple places